src/app/vault/ciphers.component.ts

```diff
@@ -44,6 +44,11 @@ export class CiphersComponent extends BaseCiphersComponent implements OnDestroy
     actionPromise: Promise<any>;
     userHasPremiumAccess = false;
 
+    organizationNames = new Map<string, string>();
+    sortedDescending = true;
+    sortBy = '' as string;
+    sortByOptions = ['name', 'owner', 'dateCreated', 'lastEdited'] as (string)[];
```

Comment:
Same here
added readonly as these values are fixed on initialization
```suggestion
    readonly sortByOptions: string[] = ['name', 'owner', 'dateCreated', 'lastEdited'];
```